src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/admin/AdminPageIT.java

```diff
@@ -0,0 +1,45 @@
+package ca.corefacility.bioinformatics.irida.ria.integration.admin;
+
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.springframework.test.context.ActiveProfiles;
+
+import ca.corefacility.bioinformatics.irida.ria.integration.AbstractIridaUIITChromeDriver;
+import ca.corefacility.bioinformatics.irida.ria.integration.pages.LoginPage;
+import ca.corefacility.bioinformatics.irida.ria.integration.pages.admin.AdminPage;
+
+import com.github.springtestdbunit.annotation.DatabaseSetup;
+
+import static junit.framework.TestCase.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+@ActiveProfiles("it")
+@DatabaseSetup("/ca/corefacility/bioinformatics/irida/ria/web/admin/AdminPageView.xml")
+public class AdminPageIT extends AbstractIridaUIITChromeDriver {
+
+	@Test
+	public void accessPageAsAdmin() {
+		LoginPage.loginAsAdmin(driver());
+		AdminPage page = AdminPage.initPage(driver());
+		assertTrue("Page title should be present", page.comparePageTitle("FUBAR - YOU MADE IT"));
+	}
+
+	@Test
+	public void accessPageFailure() {
+		LoginPage.loginAsUser(driver());
+		AdminPage page = AdminPage.initPage(driver());
+		assertFalse("Page title should not be present", page.comparePageTitle("FUBAR - YOU MADE IT"));
+	}
+
+	@Test
+	public void adminButtonVisible() {
+		LoginPage.loginAsAdmin(driver());
```

Comment:
This looks great but you can move 

`assertTrue("Admin Button should be displayed", driver().findElements(By.id("admin-panel-btn")).size() > 0);` into the accessPageAsAdmin method after the LoginPage line and remove this test.

Same applies to the test below where you can move the `assertFalse("Admin Button should not be displayed", driver().findElements(By.id("admin-panel-btn")).size() > 0);` into the accessPageFailure method after the LoginPage line and remove the test below as well

Reply:
1.  "FUBAR - YOU MADE IT" is not going to be the final title so maybe test something that will actually be there.
2.  Instead of finding by id, use a class specific for testing.  We try to prefix classes that are used with for testing with `t-` so in this case you could add the class `t-admin-panel-btn` or something like that.